### PR TITLE
Update the version pin for the `Flask-Cors` package

### PR DESCRIPTION
**Pull request by mcdonnnj**
#  #

## 🗣 Description ##

This pull request updates the version pin for the [Flask-Cors] package to include an upper bound.

## 💭 Motivation and context ##

The [4.0.0 release] removes support for Python versions < 3.8. However, the build process for releases was not updated and so the wheel released is still marked with Python 2 support (and no `Requires-Python` attribute). As a result the latest version of the package is pulled down,  which is unable to run in a Python 2 environment, and the package is unable to run.

## 🧪 Testing ##

I was able to successfully start this package through its entry point once I installed it with the updated version pin in place. It has been running in our production CyHy environment as normal.

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).

[4.0.0 release]: https://github.com/corydolphin/flask-cors/releases/tag/4.0.0
[Flask-Cors]: https://pypi.org/project/Flask-Cors/

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
👍

**Review by jsf9k — APPROVED** [review]

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "docopt >= 0.6.2",
         "flask >= 0.10.1",
         "Flask-Caching >= 1.4.0, < 1.8.0",
-        "Flask-Cors >= 2.1.0",
+        "Flask-Cors >= 2.1.0, < 4",
```
Comment (jsf9k) [inline setup.py:26]: Perhaps we should add a comment here, to explain why the `< 4` is needed.

Comment (mcdonnnj) [reply setup.py:26]: Please let me know if https://github.com/cisagov/ncats-webd/pull/38/commits/62d8efb911de4e5b80d47697eb5909607b02cc2e scratches that itch.

**Review by jsf9k — APPROVED** [review]
_Approval intensifies!!!_

**Pull request merged**